### ISSUE TITLE
[Suggestion] Change the name

### ISSUE THREAD

**Issue by Liyf4TakingLs** [TekExplorer/featherdactyl.com#42]
**For reference, what is the current latest app version?**
```
Version: v0.0.22
Build: 224
```

Currently the app name is not capitalized.

**Describe the solution you'd like**
Simply capitalize the first letter of the app.

**Additional context**
Currently it looks weird.

**Comment by drjayy** [comment]
Agreed. In advance, the old enhancement I posted in discord included shortening the name of the application on the homescreen to `Feather` as it uses up less space and doesn't directly change the whole name for the app but rather the short name on the home screen shortcut.
